extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/thing/thing-types.xml

```diff
@@ -65,13 +89,23 @@
         </config-description>
     </channel-type>
     
+    <!-- Temperature Channel -->
     <channel-type id="temperature">
         <item-type>Dimmer</item-type>
         <label>Temperature</label>
         <description>Sets the temperature of the light</description>
         <category>Temperature</category>
     </channel-type>
 
+    <!-- Infrared Channel -->
+    <channel-type id="infrared">
+        <item-type>Dimmer</item-type>
+        <label>Infrared</label>
+        <description>Sets the infrared of the light</description>
+        <category>Infrared</category>
```

Comment:
we do not have "Infrared" as a category defined in ESH, so better remove this line.